frontend/javascripts/admin/organization/organization_edit_view.tsx

```diff
@@ -99,7 +122,7 @@ class OrganizationEditView extends React.PureComponent<Props, State> {
           Attention: You will be logged out.
         </p>
       ),
-      okText: "Yes, Delete Organization now",
+      okText: "Yes, delete this organization now.",
```

Comment:
@fm3 Is this actually a hard-delete or could we restore the account upon request?

Reply:
The other question is also: What gets deleted. The orca? All annotations? All datasets? All users accounts of this organization? etc

Reply:
Sorry for the late reply, there were so many emails in this thread that I missed this.

The current implementation 
 - sets the organization table entry to isDeleted
 - sets all associated user table entries to isDeleted
 - resets the lastLoggedInIdentity in the multiUsers table for those users (so that they can still log into other orgas if allowed)
 - Note that the users and organizations being set to isDeleted will also lead to annotations and datasets not being accessible, since opening those also fetches orga/user from the db.

So I’d say yes, it can probably be restored.